Makefile

```diff
@@ -258,7 +258,7 @@ help: ## (Default) Print listing of key targets with their descriptions
 .PHONY: .gitvalidation
 .gitvalidation:
 	@echo "Validating vs commit '$(call err_if_empty,EPOCH_TEST_COMMIT)'"
-	GIT_CHECK_EXCLUDE="./vendor:./test/tools/vendor:docs/make.bat:test/buildah-bud/buildah-tests.diff:test/e2e/quadlet/remap-keep-id2.container" ./test/tools/build/git-validation -run DCO,short-subject,dangling-whitespace -range $(EPOCH_TEST_COMMIT)..$(HEAD)
+	GIT_CHECK_EXCLUDE="./vendor:./test/tools/vendor:docs/make.bat:test/buildah-bud/buildah-tests.diff:test/e2e/quadlet/remap-keep-id2.container" ./test/tools/build/git-validation -run short-subject -range $(EPOCH_TEST_COMMIT)..$(HEAD)
```

Comment:
I'm really uncomfortable with dropping the whitespace test, because I think I've seen this fail recently -- but I'm willing to see what happens. I will submit a test PR once this merges.

Reply:
I'm not 100%, but IIRC this is covered by the `pre-commit` based checks (same validate target).

Reply:
Followup: you are right. On a PR based on main after this merged, [CI hooks still detect the introduction of trailing whitespace on a line](https://cirrus-ci.com/task/5748535115644928?logs=main#L49). I have not tested for trailing blank lines at end of file.

Reply:
Thanks for double-checking @edsantiago I appreciate it.  FWIW: My main motivation is that git-validation golang tool is bespoke, old, and not well maintained.  The pre-commit tool is widely used, modular, and well maintained.